workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/GameObjectRepresentation/EntityGameObjectLinkerTests.cs

```diff
@@ -49,7 +50,7 @@ public void TearDown()
         public void LinkGameObjectToEntity_adds_SpatialOSComponent_component()
         {
             entityGameObjectLinker.LinkGameObjectToEntity(testGameObject, testEntity, testSpatialEntityId,
-                entityCommandBuffer, viewCommandBuffer);
```

Comment:
We can delete the private `entityCommandBuffer` field now as it's not used anymore.